web-app/src/routes/_layout.svelte

```diff
@@ -1,22 +1,41 @@
 <script>
-  import Nav from '../components/Nav.svelte';
-
-  export let segment;
+  // TODO: transition between pages
 </script>
 
 <style>
   main {
-    position: relative;
-    max-width: 56em;
+    display: grid;
+    grid-template:
+      "image  " min-content
+      "heading" 1fr
+      "content" 50% /
+       1fr;
+    height: 100%;
     background-color: white;
     padding: 2em;
     margin: 0 auto;
-    box-sizing: border-box;
   }
-</style>
 
-<Nav {segment}/>
+  h1 {
```

Comment:
Nit: Ideally would want to move away from element selectors for headings, since the visual hierarchy might differ from the accessibility tree/document hierarchy.

Reply:
Yeah, most of the css is pretty lazy right now, I don't think we'll end up with elements as selectors once we have time to polish